data/mods/Magiclysm/requirements/spell_components.json

```
@@ -78,5 +78,20 @@
     "type": "requirement",
     "//": "Empower your chilling touch spell with the power of ice cream!",
     "components": [ [ [ "icecream_choc", 1 ], [ "icecream", 1 ], [ "icecream_fruit", 1 ], [ "icecream_candy", 1 ] ] ]
+  },
+  {
+    "id": "spell_components_manabomb",
+    "type": "requirement",
+    "components": [ [ [ "potion_starter", 3 ] ], [ [ "rune_technomancer", 1 ] ] ]
+  },
+  {
+    "id": "spell_components_manatrap",
+    "type": "requirement",
+    "components": [ [ [ "potion_starter", 6 ], [ "alchemist_bomb", 2 ] ], [ [ "rune_technomancer", 2 ] ] ]
```

```suggestion
    "components": [ [ [ "alchemist_bomb", 2 ] ], [ [ "rune_technomancer", 1 ] ] ]
```
See comments with spell.